docker/README.md

```diff
@@ -0,0 +1,32 @@
+<!---
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License. See accompanying LICENSE file.
+-->
+# What is this?
+It is a docker image for building yosegi.
+
+# Create a docker image
+
+Please execute the command in this directory.
+
+```
+docker build -t yahoojp/yosegi_build_image .
+```
+
+# Run build
+
+You can set "BULD_WORK_DIR" and output the artifact.
+
+```
+mkdir /tmp/yosegi_build
+docker run -e BUILD_WORK_DIR=/tmp/yosegi_build -v /tmp/yosegi_build:/tmp/yosegi_build -it yahoojp/yosegi_build_image
+```
```

Comment:
Sorry, I fail this command.

[ERROR] COMPILATION ERROR :
[INFO] -------------------------------------------------------------
[ERROR] /tmp/yosegi_build/yosegi/src/main/java/jp/co/yahoo/yosegi/reader/YosegiSchemaFileReader.java:[30,20] package javax.xml.ws does not exist
[INFO] 1 error
[INFO] -------------------------------------------------------------
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  01:40 min
[INFO] Finished at: 2019-04-22T02:22:33Z
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.3:compile (default-compile) on project yosegi: Compilation failure
[ERROR] /tmp/yosegi_build/yosegi/src/main/java/jp/co/yahoo/yosegi/reader/YosegiSchemaFileReader.java:[30,20] package javax.xml.ws does not exist
[ERROR]
[ERROR] -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException

Reply:
@yohto 
This patch removes this import.
Can you check if the local code and the patch code are correct?

https://github.com/yahoojapan/yosegi/pull/45/files#diff-8cbb2886a1c46ff38786b3699fcefb31L30